apps/pie-storybook/package.json

```diff
@@ -23,6 +23,7 @@
     "@justeattakeaway/pie-link": "0.10.0",
     "@justeattakeaway/pie-modal": "0.29.0",
     "@justeattakeaway/pie-toggle-switch": "0.14.0",
+    "@justeattakeaway/pie-notification": "workspace:*",
```

Comment:
If this could be changed to `0.0.0` that'd be awesome! 🎉 

Something that needs to be changed in our generator, but hasn't been updated yet!

Reply:
Yeah I think we need to update our readme [here](https://github.com/justeattakeaway/pie/blob/main/packages/tools/generator-pie-component/README.md#setting-up-storybook) to say 

```
Add the component to the storybook package.json dependencies
"@justeattakeaway/pie-{{COMPONENT_NAME}}": "0.0.0",
```

let me know if you need any help Jose

Reply:
i updated that 🥳